aws-ssm-document/src/main/java/com/amazonaws/ssm/document/UpdateHandler.java

```diff
@@ -84,4 +97,21 @@
             throw exceptionTranslator.getCfnException(e, model.getName(), OPERATION_NAME, logger);
         }
     }
+
+    private boolean isCreateOnlyModified(final ResourceModel model, final ResourceModel previousModel) {
+        final ResourceModel comparator = ResourceModel.builder()
+                //CreatOnly Properties
+                .name(model.getName())
+                .content(model.getContent())
+                .attachments(model.getAttachments())
+                .versionName(model.getVersionName())
+                .documentType(model.getDocumentType())
+                .documentFormat(model.getDocumentFormat())
+                .targetType(model.getTargetType())
+                .requires(model.getRequires())
+                //Modifiable Properties
+                .tags(previousModel.getTags())
+                .build();
```

Comment:
are you rebuilding it just to get rid of modifiable props and why?

Reply:
Yes, this is only to compare the CreateOnly props
What I want to do here is force the modifiable properties to be equal so that we can utilize .equals rather than writing 8 conditionals

Reply:
but then why u dont strip mutable properties from previous model?

Reply:
What I am doing here is creating model that takes the createOnly properties from the desired model and the mutable properties from the previousModel
This way when we compare the 'comparator' to the previous model we ignore any differences of the mutable properties